server/src/test/java/io/crate/integrationtests/TransportSQLActionTest.java

```diff
@@ -1444,34 +1441,33 @@ public void testNumericScriptOnAllTypes() {
 
     @Test
     public void testWhereColumnEqColumnAndFunctionEqFunction() throws Exception {
-        this.setup.setUpLocations();
-        ensureYellow();
-        refresh();
+        execute("create table tbl (name text)");
+        execute("insert into tbl (name) values ('Arthur'), ('Trillian')");
+        execute("refresh table tbl");
 
-        execute("select name from locations where name = name");
-        assertThat(response.rowCount(), is(13L));
+        execute("select name from tbl where name = name");
+        assertThat(response.rowCount(), is(2L));
 
-        execute("select name from locations where substr(name, 1, 1) = substr(name, 1, 1)");
-        assertThat(response.rowCount(), is(13L));
+        execute("select name from tbl where substr(name, 1, 1) = substr(name, 1, 1)");
+        assertThat(response.rowCount(), is(2L));
     }
 
     @Test
     public void testNewColumn() throws Exception {
         execute("create table t (name string) with (number_of_replicas=0, column_policy = 'dynamic')");
-        ensureYellow();
         execute("insert into t (name, score) values ('Ford', 1.2)");
     }
 
     @Test
     public void testESGetSourceColumns() throws Exception {
-        this.setup.setUpLocations();
-        ensureYellow();
+        execute("create table tbl (id int primary key, x int)");
+        execute("insert into tbl (id, x) values (1, 1), (2, 2), (3, 3)");
         refresh();
 
-        execute("select _id, _version from locations where id=2");
-        assertNotNull(response.rows()[0][0]);
-        assertNotNull(response.rows()[0][1]);
+        execute("select _id, _version from tbl where id = 2");
+        assertThat(printedTable(response.rows()), is("2| 1\n"));
 
+        /*
```

Comment:
Should this be removed or re-enabled?

Reply:
They should be re-enabled, thanks for catching